app/src/Search/Lookup.php

```diff
@@ -116,7 +116,9 @@ protected function getURLForClass(array $searchConfig): string
     protected function staticFileExists(string $searchPath): bool
     {
         $baseDir = dirname(__DIR__, 3);
-        $path = $baseDir . '/public' . $searchPath;
+        $contents = file_get_contents($baseDir . '/doctum-conf/doctum.json');
+        $json = json_decode($contents, true);
+        $path = $baseDir . '/' . $json['paths']['www'] . $searchPath;
```

Comment:
Didn't test the below so may need some adjustment but this is how the `doctum.json` contents are intended to be accessed.
```suggestion
        $config = Config::getConfig();
        $path = $baseDir . '/' . $config['paths']['www'] . $searchPath;
```
Use `SilverStripe\ApiDocs\Data\Config`